lib/components/term.js

```diff
@@ -86,6 +89,7 @@ export default class Term extends React.PureComponent {
     this.term.open(this.termRef);
     this.term.webLinksInit();
     this.term.winptyCompatInit();
+    this.termDefaultBellSound = this.term.bellSound;
```

Comment:
Are you sure that it works?
I don't see anything in `Terminal.ts`: https://github.com/xtermjs/xterm.js/blob/master/src/Terminal.ts

Reply:
I did confirm and test throughly that this indeed works. Within `Terminal.ts` there is a constant variable declared toward the beginning of the file called `DEFAULT_OPTIONS` which contains a property `bellSound` with the value `DEFAULT_BELL_SOUND` which is imported from the xterm packages `SoundManager` class. 

I needed to offload the defaultSound value from the term instance before the method `this.term.setOption()` is called to overwrite the bellSound property so that we can restore it later. Xterm.js does not currently have logic to restore its own default value upon the bellSound being overwritten then being null or undefined later (it will just fall back on whatever was set last).